packages/rocketchat-chatops/client/views/chatops.coffee

```diff
@@ -0,0 +1 @@
+Template.chatops.helpers
```

Comment:
@marceloschmidt I mainly put this here so that when @Sing-Li went to add things everything would already be here